recipe/meta.yaml

```diff
@@ -1,88 +1,76 @@
-{% set name = "perspective" %}
-{% set version = "3.0.1" %}
+{% set name = "perspective-python" %}
```

Comment:
Undo

Reply:
OK. I figured it would be best to match the name in pypi but happy to change